explainer.md

```diff
@@ -222,11 +226,11 @@ There is a large category of attack vectors that are opened up by allowing websi
 
 The additional security-pertinent capability that this specification provides over the [file-system-access](https://github.com/WICG/file-system-access/blob/main/EXPLAINER.md#proposed-security-models) API is the ability to grant access to certain files through the operating system UI, as opposed to through a file picker shown by a web application. Any restrictions as to the files and folders that can be opened via the picker will also be applied to the files and folders opened via the operating system.
 
-There is still a risk that users may unintentionally grant a web application access to a file by opening it. However, it is generally understood that opening a file allows the application it is opened with to read and/or manipulate that file. Therefore, a user's explicit choice to open a file in an installed application, such as via an “Open with...” context menu, can be read as a sufficient signal of trust in the application.
+There is still a risk that users may unintentionally grant a web application access to a file by opening it. However, it is generally understood that opening a file allows the application it is opened with to read and/or manipulate that file. Therefore, a user's explicit choice to open a file in an installed application, such as via an “Open with...” context menu, can be read as a signal of trust in the application, and in conjunction with a permission prompt, can be viewed as sufficient trust let the web application view the file.
```

Comment:
I think this sentence should be shorter to be easier to read.  Proposal:

Therefore, a user's explicit choice to open a file in an installed application, such as via an “Open with...” context menu, can be read as a signal of trust in the application.  In conjunction with a permission prompt, this can be viewed as sufficient trust to let the web application view the file.

(note I added "to" between "trust" and "let")

Reply:
Oof thanks for the catch... looks like I didn't sufficiently check my grammar. Done.